ruleset.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0"?>
 <ruleset name="Contributte">
-    <!-- Extending rulesets -->
-    <rule ref="vendor/ninjify/coding-standard/ruleset.xml"/>
+    <!-- Contributte Coding Standard -->
+	<rule ref="./vendor/ninjify/coding-standard/contributte.xml">
+		<exclude name="SlevomatCodingStandard.Operators.DisallowIncrementAndDecrementOperators.DisallowedPostIncrementOperator"/>
```

Comment:
Is this rule used here?

Reply:
Yes, here https://github.com/contributte/console/blob/master/src/DI/ConsoleExtension.php#L107

Reply:
I could rewrite foreach to use array keys instead

Reply:
It's OK. Thanks.